cypress/support/fragments/inventory/inventoryInstances.js

```diff
@@ -92,9 +92,9 @@ export default {
       .then(() => {
         cy.getLoanTypes({ limit: 1 });
         cy.getMaterialTypes({ limit: 1 });
-        cy.getLocations({ limit: 1 });
-        cy.getHoldingTypes({ limit: 1 });
-        InventoryHoldings.getHoldingSources({ limit: 1 }).then(holdingSources => {
+        cy.getLocations({ limit: 2 });
```

Comment:
if you can use createFolioInstanceViaApi in test, let's don't make changes in this method

Reply:
fixed